blocks/library/latest-posts/index.js

```diff
@@ -54,12 +54,13 @@ registerBlockType( 'core/latest-posts', {
 
 			this.state = {
 				latestPosts: [],
+				isLoading: true,
```

Comment:
Hi, @Soean thank you for your contribution!
Would it be possible to avoid the need for an isLoading flag by setting the initial state of latestPosts to null?
This way we would be able to differentiate between the state lastest posts are not yet available and latestPosts are empty. By avoiding loading flags and just checking data availability, if later we decide to implement an offline mechanism it may make things simpler.

Reply:
Hey @jorgefilipecosta, thanks for your review. I added the changes to the PR.